README.md

```diff
@@ -0,0 +1,54 @@
+# cr-logging
+A wrapper for initialising Serilog consistently.
+
+## Configuration
+
+### Parameters
+
+All the appsettings are not mandatory; the program will not fail of these are not specified in the app.config, or their values are empty/whitepace.
+
+If an invalid value (not empty/whitespace) is entered in any of these fields, however, the program will fail.
```

Comment:
Maybe detail how it will fail, will it give an error explaining what is wrong, etc.

Reply:
From reading the code it looks like the `Setup` will throw an exception, that would be worth explaining here.